### PR TITLE
Update to Node v16 (current LTS), move to Dart Sass for SCSS compilation

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Node 16 is the current long term stable release, whereas v15 is not supported.

However… when I upgraded [CORE data design history](https://github.com/roobottom/core-design-history/pull/5/commits/8b3ca8e42dfda536dad21025f10ce1bac499f233), while `node-sass` worked locally, it caused a build failure on Netlify. I solved this by moving from `node-sass` (which uses LibSass, also no longer supported) to `sass` (which uses Dart Sass, supported).

But… the SCSS used by `govuk-frontend` uses deprecated Sass syntax (to support those still using LibSass) which throws deprecation notices when compiling styles. We can suppress these using the `--quite-deps` option.

But, but… this only works for deprecations found in dependencies. [A dependency is defined as any stylesheet loaded via a load path or importer](https://sass-lang.com/documentation/js-api/interfaces/LegacyFileOptions#quietDeps).

So… we can load the stylesheets for our components by adding the `app` folder as a load path. `@include govuk-media-query($media-type: print)` in the main stylesheet also throws a deprecation warning, but we can use `@media print` here instead.

Phew! 😅

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-15.*
+16
```
Comment (fofr) [inline .nvmrc:1]: 16.*? 16.13.0?

**Comment by fofr** [comment]
Does this also work on Heroku?

File: .nvmrc (hunk 1 above)
Comment (paulrobertlloyd) [reply .nvmrc:1]: Isn’t `16.*` the same as `16`? Do we need to pin the project to a specific minor version?

**Comment by paulrobertlloyd** [comment]
> Does this also work on Heroku?

Yes.

**Pull request merged**